README.md

```diff
@@ -114,55 +132,75 @@ The frontend library is generated in the `dist` folder, not included in the romc
 
 ## Usage <a name="usage"></a>
 
-> :information_source: romcal was re-written using ES6, and use the new import/export syntax to manage node modules. However, thanks to `esm` and `babel`, it can still be included by other non ES6 aware node modules via the normal CommonJs `require` call.
+### Installation as a node module
 
 Add romcal to your project via npm:
 
 ```bash
-$ npm install romcal
+$ npm install romcal --save
 ```
 
-Then require romcal in your node project:
+Additionally, `romcal` is also available for installtion via various "release tags" that represent different stages of development for a given version of the code.
```

Comment:
Restyle `romcal` as romcal. Do this everywhere please. :smiley:

Reply:
Done!